### PR TITLE
Add a simpler "one stop shop" for configuring and starting up the New Relic exporters

### LINKED ISSUE #76: Add a static method to configure the most standard case
If you want to configure both NR exporters, pointing at US prod, all you should need is an API Key and. service name to get started fast.

### PR DESCRIPTION
**Pull request by jkwatson**
resolves #76

### CODE REVIEW

File: opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    private boolean enableAuditLogging = false;
+    private int collectionIntervalSeconds = 5;
+
+    /**
+     * Create a new configuration. Both parameters are required.
+     *
+     * @param apiKey A valid New Relic insert key.
+     * @param serviceName The name of your service.
+     */
+    public Configuration(String apiKey, String serviceName) {
```
Comment (breedx-nr) [inline opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java:68]: Might seem a little silly, but having this covered with a test would be nice.

File: opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+import io.opentelemetry.sdk.metrics.export.MetricExporter;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+
+public class NewRelicExporters {
+
+  private static IntervalMetricReader intervalMetricReader;
+
+  /** Start up the New Relic Metric and Span exporters with the provided configuration. */
+  public static void start(Configuration configuration) {
```
Comment (breedx-nr) [inline opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java:22]: What do you think about taking it even one step further and overloading this and providing
```
public static void start(String apiKey, String appName){
    return start(new NewRelicExporters.Configuration(apiKey, "My Service Name"))
}
```
with some kind of "most common" default config?

File: opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java (hunk 1 above)
Comment (jkwatson) [reply opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java:68]: I'm cool with adding tests for the validation.  👍

File: opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java (hunk 2 above)
Comment (jkwatson) [reply opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java:22]: I think that sounds reasonable. I'll add it.

Comment (jkwatson) [reply opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java:28]: done

File: opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java (hunk 1 above)
Comment (jkwatson) [reply opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java:80]: done

**Comment by breedx-nr** [comment]
Thanks thanks.

**Pull request merged**